vars/main.yml

```diff
@@ -67,16 +67,30 @@ nms_services: [
 
 nms_module_vars: [
   acm,
+  adm,
   sm,
 ]
 
 nms_module_vals:
   acm:
     package_name: nms-api-connectivity-manager
     systemd_name: nms-acm
+    default_repo: ""
+    default_centos_yum_repo: ""
+    default_amazon_yum_repo: ""
+    default_deb_based_apt_repo: ""
+  adm:
+    package_name: nms-app-delivery-manager
+    systemd_name: nms-adm
+    default_centos_yum_repo: https://pkgs.nginx.com/adm/centos/$releasever/$basearch/
+    default_amazon_yum_repo: https://pkgs.nginx.com/adm/amzn2/$releasever/$basearch/
+    default_deb_based_apt_repo: deb [signed-by={{ nms_keyring_path | default(nms_default_keyring_path) }}] https://pkgs.nginx.com/adm/{{ ansible_distribution | lower }} {{ ansible_distribution_release }} nginx-plus
```

Comment:
huh.. good to know ADM is kind of different when it comes to installation repo...